test/realloc.c

```diff
@@ -0,0 +1,68 @@
+#include <stdlib.h>
```

Comment:
This file should not be just in the `test` folder. You should probably move it into `test/basic` I would guess.

Reply:
Deleted